iwyu_path_util.cc

```diff
@@ -190,7 +193,16 @@ string ConvertToQuotedInclude(const string& filepath) {
   for (Each<HeaderSearchPath> it(&search_paths); !it.AtEnd(); ++it) {
     // All header search paths have a trailing "/", so we'll get a perfect
     // quoted include by just stripping the prefix.
-    if (StripLeft(&path, it->path)) {
+    bool matched = StripLeft(&path, it->path);
+    if (!matched && !IsAbsolutePath(it->path)) {
+        if (isAbsolute) {
+            string absSearchPath = MakeAbsolutePath(it->path);
+            StripRight(&absSearchPath, "./");
```

Comment:
Why would an absolute path ever start with "./"? I'm guessing you put this in because you needed it, but it indicates there's something wrong with `MakeAbsolutePath`